lib/tasks/server/middleware/ember-cli:broccoli:watcher/index.js

```diff
@@ -1,25 +1,25 @@
 'use strict';
 
 var cleanBaseURL = require('clean-base-url');
-var logger       = require('heimdalljs-logger')('ember-cli:serve-files');
+var logger       = require('heimdalljs-logger')('ember-cli:broccoli:watcher');
 
 /**
- * This addon is used to serve the app (`index.html`) and
- * assets (`*.js`, `*.css`, ...) at the `baseURL` prefix.
+ * This addon is used to serve the requested assets. It runs after ember-cli:broccoli:watcher
+ * addon
```

Comment:
Is this comment right?  Isn't this the addon that installs the `ember-cli:broccoli:watcher` middleware, not one that runs after it?